breizhcrops/datasets/breizhcrops.py

```diff
@@ -221,22 +220,47 @@ def load_classmapping(self, classmapping):
         if self.verbose:
             print(f"read {self.nclasses} classes from {classmapping}")
 
-    def load(self, csv_file):
+    def load_raw(self, csv_file):
         """['B1', 'B10', 'B11', 'B12', 'B2', 'B3', 'B4', 'B5', 'B6', 'B7', 'B8',
-       'B8A', 'B9', 'QA10', 'QA20', 'QA60', 'doa', 'label', 'id']"""
+               'B8A', 'B9', 'QA10', 'QA20', 'QA60', 'doa', 'label', 'id']"""
 
         sample = pd.read_csv(os.path.join(self.csvfolder, os.path.basename(csv_file)), index_col=0).dropna()
         # convert datetime to int
         sample["doa"] = pd.to_datetime(sample["doa"]).astype(int)
         sample = sample.groupby(by="doa").first().reset_index()
-        X = np.array(sample[self.bands].values)
+
+        return sample
+
+    def load(self, csv_file):
+        sample = self.load_raw(csv_file)
+        X = np.array(sample[self.bands[1:17]].values)
 
         if np.isnan(X).any():
             t_without_nans = np.isnan(X).sum(1) > 0
             X = X[~t_without_nans]
 
         return X
 
+    def load_culturecode_and_id(self, csv_file):
+        sample = self.load_raw(csv_file)
+        X = np.array(sample.values)
+
+        if self.level == "L1C":
```

Comment:
I think this if elif clause could be replaced by
``` 
cc_index = self.bands.index("label")
id_index = self.bands.index("id")
```

in 
https://github.com/maxzoll/BreizhCrops/blob/e779cf47dbce0457dfe4a255c2d144e1778ae7d4/breizhcrops/datasets/breizhcrops.py#L63
```
self.bands = BANDS[level]
```